src/toggle-switch/toggle-switch.scss

```diff
@@ -141,6 +154,10 @@
     cursor: not-allowed;
     background-color: var(--iui-color-background-disabled);
     border-color: var(--iui-color-background-disabled);
+    @media (forced-colors: active) {
+      background-color: GrayText;
```

Comment:
I don't get why this "gray" text shows random colors in the emulated mode.

Can someone on Windows check what this looks like with the actual (not emulated) high contrast setting?

Reply:
These are taken from my Windows 10 VM running the latest version of Edge.

![Screen Shot 2022-04-12 at 3 39 00 PM](https://user-images.githubusercontent.com/849817/163040449-bc2b00ec-2ba5-4279-becd-0deeeacca48c.png)
![Screen Shot 2022-04-12 at 3 39 18 PM](https://user-images.githubusercontent.com/849817/163040657-cd8d0695-2552-48df-a65b-36a1114af15c.png)

![Screen Shot 2022-04-12 at 3 39 38 PM](https://user-images.githubusercontent.com/849817/163040669-9eb1fb3a-86cc-4a35-a632-4d248530de99.png)
![Screen Shot 2022-04-12 at 3 39 48 PM](https://user-images.githubusercontent.com/849817/163040678-33144d18-e330-4896-93c6-8b8e10ae2855.png)

Reply:
Windows toggle switch:
![image](https://user-images.githubusercontent.com/36186912/163117928-7d3b70b0-3e13-4698-a0cb-fd5ce769db8d.png)
Ours:
![image](https://user-images.githubusercontent.com/36186912/163117969-8b00304b-8491-41f6-ba78-22c0706ba10c.png)

Reply:
Active looks like disabled in our case. And handle could have brighter color.

Reply:
Vilius' image goes with this theme: 
![image](https://user-images.githubusercontent.com/37936169/163189214-0fdc0c0d-77c7-4872-93c5-b5d7700a43b1.png)
The other themes have a more noticeable difference between disabled text and selected text. The difference between text and disabled text is more different; we could use the text color instead of the selected text color?

Reply:
I changed the toggle switch handle background color & the icon fill to `HighlightText`.  There doesn't appear to be a visual change in the last 2 themes, but the first 2 themes did see a change for the better.
![Screen Shot 2022-04-15 at 3 33 52 PM](https://user-images.githubusercontent.com/849817/163624262-594bea7e-23e3-4b2b-b5dc-5543b89ce1d7.png)

Reply:
I think this looks nice. Well, nice is a strong word. I don't like high contrast, lol, makes my head hurt. But I think the change in handle color is good.